templates/step/viewimg.html.ep

```diff
@@ -1,23 +1,43 @@
 <div class="aligncenter" data-image="<%= url_for('test_img', filename => $screenshot) %>" id="step_view">
-    Candidate needle:
```

Comment:
I am not sure. I thought about this. If I do a helper I can of course put it there, too, but currently we don't have a space in the webui on wait_serial results where to put it so I would start with screenshots, add it to wait_serial later. ok?